libs/sql-schema-describer/tests/mssql/mod.rs

```diff
@@ -1,11 +1,12 @@
 use super::test_api::mssql_2019_test_api;
 use quaint::prelude::Queryable;
 use sql_schema_describer::*;
+use test_setup::TestAPIArgs;
 use tracing::debug;
 
 #[allow(dead_code)]
 pub async fn get_mssql_describer_for_schema(sql: &str, schema: &'static str) -> mssql::SqlSchemaDescriber {
-    let api = mssql_2019_test_api(schema).await;
+    let api = mssql_2019_test_api(TestAPIArgs::new(schema, 0b01000000)).await;
```

Comment:
This is not ideal, but we can figure out how to clean up afterwards as we work on sql-schema-describer test setup.

Reply:
I think there might be a way to get that value from `test_setup::connectors::CONNECTORS`, but my memory is very fuzzy.